components/Article/Page.js

```diff
@@ -143,6 +143,11 @@ const getDocument = gql`
       subscribedByMe(includeParents: true) {
         ...subInfo
       }
```

Comment:
could this already be removed?

Reply:
why should this be removed? It's used further down